test/LibrarySearch/basic.lean

```diff
@@ -220,8 +220,8 @@ lemma prime_of_prime (n : ℕ) : Prime n ↔ Nat.Prime n := by
   exact?
 
 -- Example from https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/exact.3F.20recent.20regression.3F/near/387691588
-lemma ex' (x : ℕ) (_h₁ : x = 0) (h : 2 * 2 ∣ x) : 2 ∣ x := by
-  exact? says exact dvd_of_mul_left_dvd h
+lemma ex' (x : ℕ) (_h₁ : x = 0) : 2 ∣ x := by
+  exact Nat.modEq_zero_iff_dvd.mp (congrFun (congrArg HMod.hMod _h₁) 2)
```

Comment:
@semorrison, your example broke. Should I remove it?

Reply:
Do you mean that `exact?` doesn't solve this at all anymore, or just that it produces an answer different from the `says` statement?

Reply:
If the former, we should investigate, if the later, please just update the `says` statement.

Reply:
I think it's just that reordering the lemmas has made `exact?` notice that `x = 0` directly implied `2 ∣ x`, so it does that instead of using the `2 * 2 ∣ x` hypothesis.